recipes/indexed_gzip/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "indexed_gzip" %}
+{% set version = "0.8.5" %}
+{% set sha256 = "b12408dc45ea2df7488ecfe6ca12d60c683aa46e86dda2d815007505769930ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
```

Comment:
can you this here instead of the build.sh/bat files
```
script: python -m pip install --no-deps --ignore-installed .
````

Reply:
I'd left that in separate scripts, as I wasn't sure if `$PYTHON`/`%PYTHON%` would be interpreted correctly. But it's evidently fine.